packages/templates/app/ios/Template/PolywrapDemo.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+import PolywrapClient
+
+func polywrapDemo() {
+    print("Invoking: Logging.info(...)")
+
+    let logger = Logging()
+    let logArgs = LoggingArgsLog(level: .INFO, message: "Hello there")
+    try? logger.log(args: logArgs)
+
+    // Ethereum.encodeParams
+    print("Invoking: Ethereum.encodeParams(...)")
+
+    let eth = Ethereum()
```

Comment:
why the proposed change this would change to `Ethers()`

Reply:
Fixed!